meson.build

```diff
@@ -372,17 +390,17 @@ libkmod_deps = []
 
 if zstd.found()
   libkmod_files += files('libkmod/libkmod-file-zstd.c')
-  libkmod_deps += zstd
+  libkmod_deps += zstd_dlopen ? [] : zstd
```

Comment:
This will discard both `-I/usr/include/foobar` and `-Lfoobar`. We want to use `dependency` or `partial_dependency` as mentioned above.